src/compiler/checker.ts

```diff
@@ -13027,6 +13027,10 @@ namespace ts {
             if (propName !== undefined) {
                 const prop = getPropertyOfType(objectType, propName);
                 if (prop) {
+                    if (accessNode && prop?.isDeprecated) {
+                        const deprecatedNode = accessExpression?.argumentExpression ?? (isIndexedAccessTypeNode(accessNode) ? accessNode.indexType : accessNode);
+                        errorOrSuggestion(/* isError */ false, deprecatedNode, Diagnostics._0_is_deprecated, propName as string);

```

Comment:
does this display as a suggestion? Or is it suppressed and turned into strikethrough somewhere?